core/src/components/datetime/datetime.tsx

```diff
@@ -1890,17 +1890,17 @@ export class Datetime implements ComponentInterface {
       <div class="calendar-header">
         <div class="calendar-action-buttons">
           <div class="calendar-month-year">
-            <ion-item button detail={false} lines="none" onClick={() => this.toggleMonthAndYearView()}>
-              <ion-label>
-                {getMonthAndYear(this.locale, this.workingParts)}
-                <ion-icon
-                  aria-hidden="true"
-                  icon={this.showMonthAndYear ? expandedIcon : collapsedIcon}
-                  lazy={false}
-                  flipRtl={true}
-                ></ion-icon>
-              </ion-label>
-            </ion-item>
+            <ion-button class="calendar-month-year-button" fill="clear" onClick={() => this.toggleMonthAndYearView()}>
```

Comment:
The button is no longer aligned with the next/prev buttons:

Left is `ion-button` right is `ion-item`:

![image](https://user-images.githubusercontent.com/2721089/220406104-094060c9-f71c-4f98-8900-aa39f0966052.png)